ptclient/cts/spi/dx/cl/ctMultiChangeDxa.vue

```diff
@@ -1,8 +1,8 @@
 <template>
   <el-row :gutter="12">
     <el-carousel :interval="5000" arrow="always" :autoplay="false">
-      <el-carousel-item v-for="(dxList, index) in sliderSet" :key="`dxCarousel-${index}`">
-        <el-col v-for="(dx, index) in dxList" :key="index" :span="8">
+      <el-carousel-item v-for="(dxlist, pIndex) in sliderSet" :key="`dxCarousel-${pIndex}`">
+        <el-col v-for="(dx, cIndex) in dxlist" :key="cIndex" :span="8">
```

Comment:
There are overlapping `index` variables here. When nesting, these should be unique declarations. For the sake of simplicity, `pIndex` and `cIndex` are used which indicate Parent and Child indices.